source/_integrations/mastodon.markdown

```diff
@@ -59,4 +59,68 @@ base_url:
   default: https://mastodon.social
 {% endconfiguration %}
 
-To use notifications, please see the [getting started with automation page](/getting-started/automation/).
+### Usage
+
+Mastodon is a notify platform, and can be used by calling notify service [as described here](/integrations/notify/). It will toot message using 
```

Comment:
```suggestion
Mastodon is a notify platform, and can be used by calling notify service as described in the [notify documentation](/integrations/notify/). It will toot messages using 
```